### ISSUE TITLE
Predictions on LightGCN change each time they are executed

### ISSUE THREAD

**Issue by JavierSanzCruza** [beta-team/beta-recsys#421]
### Describe the bug
Following the implementation of the LightGCN model implemented in this framework, I obtain different prediction scores each time I execute the `predict` function. I am not retraining the model (I am only re-executing the `predict` method), so I expected the scores to be the same after each execution.

This is likely due to the dropout layer in the `forward` function, which introduces randomization to the prediction scores when it is used in prediction. 

### To Reproduce

I include code with an example:

```
from beta_rec.datasets.movielens import Movielens_100k
from beta_rec.data import BaseData
dataset = Movielens_100k()
split_dataset = dataset.load_temporal_split(n_test=1,n_negative=100)
data =  BaseData(split_dataset)

config = {
    "config_file":"./configs/lightgcn.json"
}

from beta_rec.recommenders import LightGCN
model = LightGCN(config)
model.train(data)
model.predict(data.train)
```

Each time the `predict` function is called, it returns different values, even when the dataframe it receives is the same.

### Describe your attempts
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this is not a duplicate issue

I also checked other model (MatrixFactorization) and did not observe this issue for that model.

### Context
 - **OS** [e.g. Windows 10, macOS 10.14]: 
 - **Hardware** [e.g. CPU only, GTX 1080 Ti]: 
 - **Environment** [e.g. Pytorch 1.4, Tensorflow 2.0]: Python 3.8, Pytorch 1.10, Pandas 1.19.5

The Beta Recsys version that I used was 0.3.2 (I installed it with pip)

### Additional Information
This is likely due to the dropout layer in the `forward` function, which introduces randomization to the prediction scores when it is used in prediction.

**Comment by TedSIWEILIU** [comment]
Thanks for pointing out the issue. I have fixed this bug by changing the forward function to activate the dropout only for each training forward. We will release the new code soon. You may want to try the new version and close this issue if it's fixed.

Thanks for your contribution.

**Comment by TedSIWEILIU** [comment]
This issue has been resolved by @JavierSanzCruza. Check his latest Pull Request for the change. Thanks, Javier!
